stackImage.py

```diff
@@ -9,24 +9,28 @@
 # Save folder to save images.
 saveFolder = 'save-folder'
 
-def stackImage(currentImage, overlayImage):
-    """
-    stackImage takes two arguments, args[0] is the base image and args[1] is the image to superimpose over the base.
-    This function will take two images, and return one image.
-    """
-    fileInfo = currentImage.split('/')
-
-    print 'stackImage started...'
-    # Base image
-    print 'DEBUG backround image = ' + currentImage
-    background = Image.open(open(currentImage,'rb'))
-    print currentImage + ' is your base image'
-
-    # Overlay image
-    overlay = Image.open(overlayImage)
-
-    background.paste(overlay, (0, 0), overlay)
-    background.save(saveFolder + '/' + prefixText + fileInfo[2])
-
-    print fileInfo[2] + ' saved to save-folder.'
-    print 'stackImage completed...'
+def stackImage(currentImage, overlayImage, processAllPictures):
+	"""
+	stackImage takes two arguments, args[0] is the base image and args[1] is the image to superimpose over the base.
```

Comment:
This no longer looks accurate @jwu910 , from what I'm reading you now take 3 arguments and args[2] is a flag of sorts